src/veh_type.h

```diff
@@ -143,13 +143,18 @@ struct transform_terrain_data {
 };
 
 const std::vector<std::pair<std::string, translation>> vpart_variants = {
-    { "cover", to_translation( "vpart_variants", "Cover" ) },
+    { "cover_left", to_translation( "vpart_variants", "Cover Left" ) },
+    { "cover_right", to_translation( "vpart_variants", "Cover Right" ) },
+	{ "hatch_wheel_left", to_translation( "vpart_variants", "Hatchback Wheel Left" ) },
```

Comment:
```suggestion
    { "hatch_wheel_left", to_translation( "vpart_variants", "Hatchback Wheel Left" ) },
```
This was a tab instead of spaces.